### PR TITLE
Deploy docs to new website

### PR DESCRIPTION
**Pull request by johnspackman**

### CODE REVIEW

**Review by derrell — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]
I would prefer a GitHub Action based new soloution.
This should be just a quick hack

**Review by cboulanger — CHANGES_REQUESTED** [review]
Thinking about it, let's not use the travis workflow anymore and create a dedicated github action workflow instead . See #9953

File: .travis/deploy_website [hunk 1]
```diff
@@ -15,7 +15,7 @@ git config user.name "Travis CI deploy"
 git config user.email "no-reply@qooxdoo.org"
 git config push.default simple
 
-GIT_IO_URL="https://$GH_TOKEN@github.com/qooxdoo/qooxdoo.github.io.git"
+GIT_IO_URL="https://$GH_TOKEN@github.com/qooxdoo/website.io.git"
```
Comment (oetiker) [inline .travis/deploy_website:18]: that seems odd ... 

```suggestion
GIT_IO_URL="https://$GH_TOKEN@github.com/qooxdoo/website.git"
```

**Review by oetiker — CHANGES_REQUESTED** [review]

**Comment by cboulanger** [comment]
I think this PR is moot because we use a github workflow now. Can we close @johnspackman ?

**Pull request closed without merging**